template/src/screens/DemoScreen.tsx

```diff
@@ -29,33 +21,13 @@ interface DemoScreenProps {
 }
 
 const DemoScreen = ({ navigation }: DemoScreenProps) => {
-  const [isLogoutLoading, setIsLogoutLoading] = useState(false)
   const counter = useAppSelector(selectCounter)
-  const comicRequest = useAppSelector(selectComic)
   const dispatch = useAppDispatch()
   const { t } = useTranslation()
 
-  useEffect(() => {
-    if (isNotRequested(comicRequest)) {
-      dispatch(getLatestComicAsync())
```

Comment:
No need to explicitly call initial fetch. It's handled by default by TanStack Query inside `useLatestComicQuery`